ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -2495,6 +2495,21 @@ tests:
     - ref: olmv1-disable-default-clustercatalogs
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn
+- as: baremetalds-ipi-ovn-lvms-f14-disasterrecovery
+  cluster: build05
+  cron: 1 13 5,25 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      FILTERS_ADDITIONAL: ""
+      LVM_OPERATOR_SUB_CHANNEL: stable-4.19
+      LVM_OPERATOR_SUB_SOURCE: qe-app-registry
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+      TEST_SCENARIOS: DR_Testing
+    test:
+    - ref: openshift-extended-test
```

Comment:
For this test design, only the DR cases are executed. 
We already have `openshift-e2e-test-qe-disasterrecovery` https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/test/qe/disasterrecovery/openshift-e2e-test-qe-disasterrecovery-chain.yaml#L2, run `openshift-extended-test-disasterrecovery` including DR cases, then `openshift-extended-test`.
Also we have chain `openshift-e2e-test-qe-disasterrecovery-sanity`, https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/test/qe/disasterrecovery/sanity/openshift-e2e-test-qe-disasterrecovery-sanity-chain.yaml#L2,  run `openshift-extended-test-disasterrecovery` including DR cases, then `openshift-extended-test` with Level0.

Reply:
Updated, thanks